daft/runners/ray_tracing.py

```diff
@@ -617,7 +617,7 @@ class _RayRunnableFunctionWrapper:
     task: PartitionTask[Any]
 
     def options(self, *args: Any, **kwargs: Any) -> _RayRunnableFunctionWrapper:
-        return dataclasses.replace(self, f=self.f.options(*args, **kwargs))  # type: ignore[no-untyped-call]
+        return dataclasses.replace(self, f=self.f.options(*args, **kwargs))
```

Comment:
Btw a little further down we have an `ray.ObjectRef[Any]` which is incorrect right?

Reply:
Actually this happens a number of times throughout the codebase, which might need to be fixed if we confirm it's incorrect